primitives/src/pcs/transcript.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
```

Comment:
Why do we need transcript in the PCS crate?

Reply:
In `jf-plonk` there's a more general transcript implementation. However some transcript sub-modules depend on things defined in `jf-plonk` so we cannot use it in `jf-primitives`.
We could put a TODO here.

Reply:
I see. On the other hand, where is the place that we use transcript in PCS? I'm wondering if we can remove the use of transcript in PCS.

Reply:
The batching part. Right now only for multilinear settings.

Reply:
I see, thanks. Let's put a TODO here then.